mta-st.yaml

```diff
@@ -0,0 +1,95 @@
+_schema-version: '2.1'
+ID: bookshop-java-public
+version: 1.0.0
+description: "Bookshop CAP Java Project with UI"
+parameters:
+  enable-parallel-deployments: true
+modules:
+# --------------------- SERVER MODULE ------------------------
+  - name: bookshop-java-public-srv
+# ------------------------------------------------------------
+    type: java
+    path: srv
+    parameters:
+      memory: 1024M
+      disk-quota: 256M
+      buildpack: sap_java_buildpack
+    properties:
+      SPRING_PROFILES_ACTIVE: cloud
+    build-parameters:
+      builder: custom
+      commands:
+        - mvn clean package -DskipTests=true
+      build-result: target/*-exec.jar
+    requires:
+      - name: bookshop-java-public-hdi-container
+      - name: bookshop-java-public-uaa
+      - name: hdi_credentials
+        properties:
+          CDS_MULTITENANCY_DEPLOYER_USER: ~{user}
+          CDS_MULTITENANCY_DEPLOYER_PASSWORD: ~{password}
+    provides:
+      - name: srv-url
+        properties:
+          url: '${default-url}'
+# --------------------- DB MODULE ---------------------------
+  - name: bookshop-java-public-db
+# -----------------------------------------------------------
+    type: nodejs
+    path: db
+    parameters:
+      memory: 256M
+      disk-quota: 512M
+      no-route: true
+      no-start: true
+      tasks:    # execute hdi deployer on app creation/update
+      - name: hdi-deploy
+        command: npm run start
+    requires:
+      # Set user and password to secure rest API via basic authentication
+      - name: bookshop-java-public-hdi-container
+      - name: hdi_credentials
+        properties:
+          hdi_dynamic_deploy_user: ~{user}
```

Comment:
this is a simple HDI container, right? no need for dynamic deployer user and password.

Reply:
Correct. I removed it.